dash.js

```diff
@@ -851,8 +851,8 @@ var DockDash = GObject.registerClass({
         let newIndex = 0;
         let oldIndex = 0;
         while (newIndex < newApps.length || oldIndex < oldApps.length) {
-            const oldApp = oldApps.at(oldIndex);
-            const newApp = newApps.at(newIndex);
+            const oldApp = oldApps.length > oldIndex ? oldApps[oldIndex] : null;
+            const newApp = newApps.length > newIndex ? newApps[newIndex] : null;
```

Comment:
This should be fine as well:

```suggestion
            const oldApp = oldApps[oldIndex];
            const newApp = newApps[newIndex];
```

Reply:
let's ignore this, it may still lead to warning in old gjs versions, not an issue, but let's keep the old verbose code :(